ci/priv-integration.sh

```diff
@@ -85,6 +85,64 @@ systemd-run -dP --wait podman build -t localhost/fcos-derived .
 derived_img=oci:/var/tmp/derived.oci
 systemd-run -dP --wait skopeo copy containers-storage:localhost/fcos-derived "${derived_img}"
 
+# Verify policy
+cat > /etc/containers/registries.d/default.yaml << EOF
+docker:
+  quay.io/rh_ee_rsaini/coreos:
+    use-sigstore-attachments: true
+EOF
+
+cat > /etc/containers/policy.json << EOF
+{
+    "default": [
+        {
+            "type": "reject"
+        }
+    ],
+    "transports": {
+        "docker": {
+            "quay.io/fedora/fedora-coreos": [
+                {
+                    "type": "insecureAcceptAnything"
+                }
+            ],
+            "quay.io/rh_ee_rsaini/coreos": [
+                {
+                    "type": "sigstoreSigned",
+                    "keyPath": "/etc/pki/containers/fcos.pub",
```

Comment:
While this doesn't matter functionally, I'd write the key *first* before writing this policy file.